typescript.ts

```diff
@@ -17,6 +17,7 @@ export = {
   ],
 
   rules: {
+    'no-shadow': 'off', // Disable basic rule in favor of `@typescript-eslint/no-shadow`
```

Comment:
@giotramu please add a "separator" comment (`// --- ES`) before and an empty line after the `no-shadow` rule

Reply:
@StefanoMagrassi now it should be fine